_data/simple-icons.json

```diff
@@ -1030,6 +1030,11 @@
             "hex": "189EFC",
             "source": "https://github.com/tootsuite/mastodon/blob/master/app/assets/images/logo.svg"
         },
+        {
+            "title": "Mathematica",
```

Comment:
I believe the full title is `"Wolfram Mathematica"`, with Mathematica being a shorthand that is often used (even by Wolfram themselves).